automation/e2e-deploy-resources.sh

```diff
@@ -40,7 +40,7 @@ kubectl apply -f "https://github.com/kubevirt/kubevirt/releases/download/${KUBEV
 
 kubectl apply -f "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml"
 
-kubectl patch kubevirt kubevirt -n kubevirt --type merge -p '{"spec":{"configuration":{"developerConfiguration":{"featureGates": ["DataVolumes"]}}}}'
+kubectl patch kubevirt kubevirt -n kubevirt --type merge -p '{"spec":{"configuration":{"developerConfiguration":{"featureGates": ["DataVolumes", "VMExport"]}}}}'
```

Comment:
`DataVolumes` FG can be dropped I guess.

Reply:
Yep just want to get confirmation from @ksimon1 about it, so this PR won't do any other changes except adding new tests.

Reply:
yes, it can be dropped

Reply:
I dropped it.